packages/prisma/migrations/20230523102615_added_booker_layouts/migration.sql

```diff
@@ -0,0 +1,5 @@
+-- AlterTable
```

Comment:
Add bookerlayouts to both event type as well as users. I think the json format makes the most sense here, right? 
See zod file a few files down for the format.